src/magic_search.html

```diff
@@ -10,18 +11,19 @@
                         {{ facet.label[0] }}:<b>{{ facet.label[1] }}</b>
                     </span>
                     <a class="remove" ng-click="removeFacet($index, $event)" title="{{ strings.remove }}">
-                        <i class="fi-x"></i>
+                        <i class="fi-x fa fa-times"></i>
                     </a>
                 </span>
             </span>
             <span id="search-selected" class="label" ng-cloak="" ng-show="facetSelected">
                 {{ facetSelected.label[0] }}:
             </span>
-            <div id="search-entry">
-                <input id="search-input" type="text" data-dropdown="facet-drop"
```

Comment:
data-dropdown is required for foundation dropdowns. If we want to retain foundation dropdown triggers as-is, we may need to have separate html files for foundation and bootstrap.